spec/logSearch_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rspec'
+require_relative '../lib/logSearch'
```

Comment:
File names for specs also follow the snake case convention 'log_search_spec'.  (class_file_name_spec)